src/authorizations/components/redesigned/GenerateTokenDropdown.tsx

```diff
@@ -27,6 +29,7 @@ const GenerateTokenDropdown: FC<ReduxProps & GenerateTokenProps> = ({
   }
 
   const handleCustomApi = () => {
+    getResources()
```

Comment:
I think the error should be handled here. If `handleCustomApi` was made `async`, this call can get wrapped in a `try/catch` block and in the `catch` block, we can fire off a notification saying "there was an error, please reload" or something along those lines.

As it stands now, if the fetch fails, it will log in the console and not let the user know anything has happened.